CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing guidelines
+
+Thank you for your interest in the project. We look forward to your
+contribution. In order to make the process as fast and streamlined as possible,
+here is a set of guidelines we recommend you follow.
+
+## Reporting issues
+
+We do our best to ensure bugs don't creep up in our releases, but some may
+still slip through. If you encounter one while using docker-py, please create
+an issue [in the tracker](https://github.com/docker/docker-py/issues/new) with
+the following information:
+
+- docker-py version, docker version and python version
+```
+pip freeze | grep docker-py && python --version && docker version
+```
+- The issue you're encountering including a stacktrace if applicable
+- If possible, steps or a code snippet to reproduce the issue
+
+To save yourself time, please be sure to check our
+[documentation](http://docker-py.readthedocs.org/) and use the
+[search function](https://github.com/docker/docker-py/search) to find out if
+it has already been addressed, or is currently being looked at.
+
+## Submitting pull requests
+
+Do you have a fix for an existing issue, or want to add a new functionality
+to docker-py? We happily welcome pull requests. Here are a few tips to make
+the review process easier on both the maintainers and yourself.
+
+### 1. Sign your commits
+
+Please refer to the ["Sign your work"](https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work)
+paragraph in the Docker contribution guidelines.
+
+### 2. Make sure tests pass
```

Comment:
I guess that's fair. I considered it, but since we're not really in the spotlight when contributor events happen I thought it wasn't necessary. 

But it would surely make the project more welcoming to a broader range of people.